Change instances of Engine to Graph Manager (#3176)
apollographql/apollo-server: docs/source/api/apollo-server.md

*  `apiKey`: string __(required)__

  API key for the service. Obtain an API key from
  [Graph Manager](https://engine.apollographql.com) by logging in and creating
  a service. You can also specify an API key with the `ENGINE_API_KEY`

>   environment variable. This option takes precedence over the environment variable.

Reviewer: This reads a little funny, though I understand.

>You can also specify an API key with the `ENGINE_API_KEY`environment variable. This option takes precedence over the environment variable.

```suggestion
  environment variable, however the `apiKey` option takes precedence.
```

*  `reportIntervalMs`: number

   How often to send reports to Graph Manager, in milliseconds. We'll also send reports
   when the report reaches a size threshold specified by `maxUncompressedReportSize`.

*  `maxUncompressedReportSize`: number

   In addition to interval-based reporting, Apollo Server sends a report to
   Graph Manager whenever the report's size exceeds this value in
   bytes (default: 4MB). Note that this is a rough limit. The size of the

>    report's header and some other top-level bytes are ignored. We just add up the lengths of the serialized traces and signatures.

Reviewer: ```suggestion
   report's header and some other top-level bytes are ignored. The report size is limited to the sum of the serialized traces and signatures.
```

*  `rewriteError`: (err: GraphQLError) => GraphQLError | null

   By default, all errors are reported to Apollo Graph Manager.  This function
   can be used to exclude specific errors from being reported.  This function
   receives a copy of the `GraphQLError` and can manipulate it for the
   purposes of Graph Manager reporting.  The modified error (e.g., after changing
   the `err.message` property) should be returned or the function should return
   an explicit `null` to avoid reporting the error entirely.  It is not
   permissible to return `undefined`. Note that most `GraphQLError` fields,

>    like `path`, will get copied from the original error to the new error: this

Reviewer: ```suggestion
   like `path`, will be copied from the original error to the new error: this
```

   Creates a client context(ClientInfo) based on the request pipeline's
   context, which contains values like the request, response, cache, and

>    context. This generated client information will be provided to Apollo

Reviewer: We seem to just be referring to 'Graph Manager' rather than 'Apollo Graph Manager' in most cases

```suggestion
   context. This generated client information will be provided to
```
